### PR TITLE
fix(android): set background color for chip text in light mode

### PR DESCRIPTION
**Pull request by dsilva609**
## ✏️ Description
- fixes an issue on android where the background color of chip text doesn't match

## 🔬 To Test
1. Go to explore tab
2. Scroll down to the bottom with the campus tags
3. The text shouldn't have a white background inside the buttons
## 
📸 Screenshots
Before
image
After
image

## 📝 Checklist

- [x] I have performed a self-review of my code
- [x] I have tested my code works in the project (nothing breaks)
- [x] I have commented my code in hard-to-understand areas
- [x] New and existing tests and linting pass locally with my changes
- [x] I have made corresponding changes to the documentation (new envs?)
- [x] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

File: templates/mobile/src/features/ChipListFeatureConnected/ChipListFeature/index.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const ChipText = styled(({ theme }) => {
 
 const ChipButton = styled(() => ({
   borderRadius: 8,
+  backgroundColor: Appearance.getColorScheme() === 'light' ? 'lightgrey' : '',
```
Comment (calebpanza) [inline templates/mobile/src/features/ChipListFeatureConnected/ChipListFeature/index.js:33]: I'm concerned that this is actually an error on the `Chip` component and not just the `ChipListFeature`. We also shouldn't use system colors in instances like this, but rather the Apollos theme colors

**Review by calebpanza — CHANGES_REQUESTED** [review]

File: templates/mobile/src/features/ChipListFeatureConnected/ChipListFeature/index.js (hunk 1 above)
Comment (dsilva609) [reply templates/mobile/src/features/ChipListFeatureConnected/ChipListFeature/index.js]: Moved this over to `Chip` and it seems more consistent now.

image

Comment (calebpanza) [reply templates/mobile/src/features/ChipListFeatureConnected/ChipListFeature/index.js]: This looks a lot better!

**Comment by calebpanza** [comment]
Snaps are out of date

**Comment by dsilva609** [comment]
> Looks good to me! I also noticed that the horizontal scrolling indicator is visible on the Chip List. Not required for this PR to get merged, but it would be nice to get that hidden on this component if you wanted to add that real quick

@calebpanza just pushed changes to hide that on the container

**Pull request merged**